src/component/common/InstanceStatus/InstanceStatus.tsx

```diff
@@ -14,9 +39,40 @@ export const InstanceStatus: FC = ({ children }) => {
                     <InstanceStatusBarMemo instanceStatus={instanceStatus!} />
                 )}
             />
+            <Dialogue
```

Comment:
This Dialogue should only display options to extend if the user is an admin. Otherwise we only want to show options to close the dialogue. 

In addition, if `instanceState.trialExtended >= 1` we don't want to show the option to extend.